hid-xpadneo/src/hid-xpadneo.c

```diff
@@ -402,9 +401,9 @@ static int xpadneo_get_battery_property(struct power_supply *psy,
 		    || xdata->psy_desc.type == POWER_SUPPLY_TYPE_UNKNOWN)
 			val->intval = POWER_SUPPLY_CAPACITY_LEVEL_UNKNOWN;
 		else if (XPADNEO_BATTERY_MODE(flags)
-			 || XPADNEO_BATTERY_CHARGING(flags)) {
+			 || XPADNEO_BATTERY_CHARGING(flags))
 			val->intval = capacity_level_map[level];
```

Comment:
I'm not sure about this one... It looks like the kernel formatting script automatically added the braces here. Could you check by running `indent` on the source? There's an `.ident.pro` profile which is used when you run `indent` from the project root (aka `indent -v hid-xpadneo/src/hid-xpadneo.c`).

Reply:
But I agree that "} else" looks ugly. I'm pretty sure that `indent` did this.

Reply:
Running indent just put spaces between the curly braces on lines 817, 818, 821 and 824

Reply:
```
$ indent --version
GNU indent 2.2.12
```

Reply:
Okay, let's pick this change and I'll cross-check with my local indent version.

Reply:
I got some time to remember... Yes, I think ident was confused about that part and put spaces into strange places. Maybe that should be refactored. I'll probably look at your PR tomorrow. If you have any updates on this, feel free to rebase and push.